inference_segmentation.py

```diff
@@ -179,13 +180,20 @@ def eval_batch_generator(
         inputs = batch["image"].to(device)
         with torch.no_grad(), autocast(device_type=device.type):
             outputs = model(inputs)
-        if single_class_mode:
-            outputs = torch.sigmoid(outputs)
+        if check_in:
+            # This option is activate only when the code check for the pourcentage of Ram available,
+            # avoiding all the useless permutation since we dont use the output data.
+            batch_output['data'] = outputs
```

Comment:
Second thought: rather than add an if statement here to accelarate the auto_batch_size_finder, shouldn't we just try to fix the batch size in production and skip this batch size finding, knowing we have more GPU RAM than we need?